exporter/clickhousetracesexporter/migrations/000014_add_usage_explorer_mv.up.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS signoz_traces.usage_explorer (
+  timestamp DateTime64(9) CODEC(DoubleDelta, LZ4),
+  service_name LowCardinality(String) CODEC(ZSTD(1)),
+  count UInt64 CODEC(T64, ZSTD(1))
+) ENGINE SummingMergeTree()
+PARTITION BY toDate(timestamp)
+ORDER BY (timestamp, service_name);
```

Comment:
@makeavish high cardinality sorting key before low cardinality is not recommended IMO

Reply:
But in all of the cases we are using timestamp as filter and service filter is secondary filter.

Reply:
okay..then this should be fine. A query with `service_name` and without `timestamp` would perform poor in the current schema